src/Compiler/NodeVisitor/ChangeFiltersNodeVisitor.php

```diff
@@ -28,6 +29,10 @@
 use PhpParser\Node\VariadicPlaceholder;
 use PhpParser\NodeVisitorAbstract;
 use PHPStan\BetterReflection\BetterReflection;
+use PHPStan\BetterReflection\Reflection\ReflectionFunction;
```

Comment:
We try to replace direct use of BetterREflection with PHPStan ReflectionProvider when possible for performance reasons

Reply:
In this file, BetterReflection was used so I continue with it. I don’t want to mix things in pull request. For replacing there is another issue, when it can be done as the only thing at that PR.

Reply:
if it is already used then keep it we will repalce it later